install/heroku_postgres/01_deploy_the_collector.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: 'Step 1: Deploy the Collector'
+install_track_title: Installation Guide (Heroku Postgres)
+backlink_href: /docs/install
+backlink_title: 'Installation Guide'
+---
+
+import imgHerokuPaidDyno from "../../images/heroku_paid_dyno.png"
+import { collectorAppName } from "./util";
+
+export const APIKey = ({ apiKey }) => <code>{apiKey}</code>;
+
+export const CollectorApp = ({ organizationSlug }) => (
+  <code>{collectorAppName(organizationSlug)}</code>
+);
+
+In order to monitor a Heroku Postgres database you need to run the pganalyze
+collector as an app inside your Heroku account. Through Heroku's add-on attachment system you then attach all databases that should be monitored.
+
+## Deploy collector
+
+First, deploy the collector to your own Heroku account with your pganalyze API key <APIKey apiKey={props.apiKey} />:
```

Comment:
We should probably have separate logic here for in-app (where we have an API key, vs public docs):

<img width="675" alt="Screen Shot 2021-01-16 at 2 35 16 PM" src="https://user-images.githubusercontent.com/7227/104824608-0dd55700-5808-11eb-8dbb-59f80b99ee4f.png">

(note the small empty `<code>` block)